requirements/build.txt

```diff
@@ -309,3 +299,16 @@ docutils==0.14 \
     --hash=sha256:7a4bd47eaf6596e1295ecb11361139febe29b084a87bf005bf899f9a42edc3c6 \
     --hash=sha256:02aec4bd92ab067f6ff27a38a38a41173bf01bed8f89157768c1573f53e474a6 \
     --hash=sha256:51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274
+mozilla-django-oidc==0.3.1 \
+    --hash=sha256:f906d4ea1879ee588da4a4369013ade19101624516b977fc96b271144123b446 \
+    --hash=sha256:59abd3941ba3471a7baef18a0f704868821cc48ccedd6469fa9e1d08fdc737f3
+python-jose==1.3.2 \
+    --hash=sha256:35eca894ab91db1774251296949679396c46d6bc506ea03804b8f7a7d0204392 \
+    --hash=sha256:968254f57ccd0fc99ab9557f82b90a5b23ccaf3716e8817d8edaa9f21c21bb2d
+pycrypto==2.6.1 \
+    --hash=sha256:f2ce1e989b272cfcb677616763e0a2e7ec659effa67a88aa92b3a65528f60a3c
```

Comment:
Not a huge fan of including pycrypto here since it has been unmaintained for a while now. Is there no way to use cryptography instead?

Reply:
Looks like `python-jose` depends on either `pycrypto` or https://pypi.python.org/pypi/pycryptodome .

Reply:
Oops, upon further review looks like `python-jose` refuses to build without `pycrypto` unless you're using PyPy.

Reply:
Dang it, this sucks, but I guess we can't help it, unless we get python-jose moved to cryptography. Having to maintain a project with pycrypto is a big risk IMO.

Reply:
Seems like we're not the only ones: https://github.com/capless/warrant/issues/31, https://github.com/mpdavis/python-jose/issues/41 and most notably: https://blog.sqreen.io/stop-using-pycrypto-use-pycryptodome/